plugin/teksi_wastewater/interlis/interlis_importer_exporter.py

```diff
@@ -153,6 +153,8 @@ def interlis_export(
         export_models,
         logs_next_to_file=True,
         limit_to_selection=False,
+        # neu export_orientation
+        export_orientation=90,
```

Comment:
```suggestion
        export_orientation=90.0,
```
So it is initialized as float